### PR TITLE
Fixing the way to package the lib & Fix the copyright of the license

### PR DESCRIPTION
**Pull request by Nexucis**

### CODE REVIEW

**Comment by Nexucis** [comment]
this PR is mainly because the job that is releasing the lib failed and so far no idea why: https://app.circleci.com/pipelines/github/promlabs/lezer-promql/117/workflows/a14933e0-ad69-4dce-9fd2-d7f39a1a7f1a/jobs/137

I think it's a bit cleaner like that. The idea is that the job triggered by the release won't do anything than pushing the package to npm. Other step (like copying file) should move the build job to detect immediatly if it is working or not.

File: LICENSE [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 PromLabs GmbH
```
Comment (juliusv) [inline LICENSE]: This line is part of the instructions on how to use the license (e.g. in code files), not the actual copyright line for this library itself. We had the same situation in Prometheus :)

So far I haven't put the license text at the top of all files of the repo yet, I guess we could do that at some point.

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
```
Comment (juliusv) [inline build.sh:2]: For bonus points, add this at the top of any bash script:

```bash
set -ex
```

(see https://www.peterbe.com/plog/set-ex)

File: build.sh [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+lezer-generator src/promql.grammar -o src/parser
+
+cat src/parser.terms.js >> src/parser.js
+
+bash ./generate-types.sh
+
+rollup -c
+
+# Finally copy some useful file for documentation purpose
```
Comment (juliusv) [inline build.sh]:
```suggestion
# Finally, copy some useful files into the distribution folder for documentation purposes.
```

File: LICENSE (hunk 1 above)
Comment (Nexucis) [reply LICENSE]: mmm ok. Yeah I guess we can add the license on top of each file

**Comment by Nexucis** [comment]
should good for a second turn of review @juliusv :)

**Comment by juliusv** [comment]
:+1: Thanks :)

**Pull request merged**